cmd/diskmaker/main.go

```diff
@@ -18,8 +18,8 @@ var managerCmd = &cobra.Command{
 }
 var lvDaemonCmd = &cobra.Command{
 	Use:   "lv-controller",
-	Short: "Used to start an instance of the controller for a specific LocalVolume CR",
-	RunE:  startDiskMaker,
+	Short: "Used to start the controller-runtime manager that owns the LocalVolume controller",
+	RunE:  startManager,
```

Comment:
if it's using the same function as lv-manager, no need to add a new command, just change the command invoked in `generateDiskMakerDaemonSet `to `lv-manager`

Reply:
However, this will also start a lvset controller on every diskmaker daemonset spawned by localvolume_controller. Maybe we should just include the localvolume nodeSelectors in `/home/tpjsm/go/src/github.com/openshift/local-storage-operator/pkg/controller/nodedaemon/reconcile.go`/`lvSets, tolerations, ownerRefs, nodeSelector, err := r.aggregateDeamonInfo(request)`

the tolerations, ownerRefs, and nodeSelector should incorporate info from all localvolumes like it currently does for localvolumesets

Reply:
@rohantmp , as the `LocalVolumeSet`  has the `ProvisionerConfigMap, DiskMaker DaemonSet` in `nodedaemon controller` , the `ProvisionerConfigMap, DiskMaker DaemonSet` for LocalVolume should also move to nodedaemon, so that there is a single daemonset for all `LocalVolume CRs`. This will also require some functional tests to modify. Do you want me to do this in the same PR, or open a different one?

Reply:
No need to change the provisioner configmap! And leave LocalVolume to create one local-static-provisioner per LocalVolume. We can fix that in a follow up where I integrate static-provisioner to LocalVolume

Reply:
Thanks! Will update the PR to only change to the single ` Diskmaker Daemonset ` for LocalVolume.